src/idlcxx/src/streamer_generator.c

```diff
@@ -721,9 +783,7 @@ idl_retcode_t check_alignment(context_t* ctx, int bytewidth, bool is_key)
       format_write_stream(0, ctx, false, "size_t ");
       ctx->streamer_funcs.alignmentpresent = true;
     }
-    format_write_stream(0, ctx, false, alignmentbytes, ctx->depth);
-    format_write_stream(0, ctx, false, " = ");
-
+    format_write_stream(0, ctx, false, alignmentbytes " = ", ctx->depth);
```

Comment:
From the context I could make out `alignmentbytes` was a macro. I won't require you to fix it right now, but perhaps it's good to know that macro's are always uppercase within the Eclipse Cyclone DDS project and is common practice in a lot of other code bases.

Reply:
You are correct.
I should keep this in mind when go through an iteration of cleanup and tidying.